runxtb.sh

```diff
@@ -614,7 +614,11 @@ write_submit_script ()
         fi
         echo "#SBATCH --export=NONE" >&9
       fi
-      submit_commandline=( "srun" "$xtb_callname" "${xtb_commands[@]}" )
+      if [[ "${xtb_callname}" == xtb ]]; then
+        submit_commandline=( "srun" "$xtb_callname" "${xtb_commands[@]}" )
+      elif [[ "${xtb_callname}" == crest ]]; then
+        submit_commandline=( "srun" "$xtb_callname" "${xtb_commands[@]}" "-T" "${requested_numCPU}" )
+      fi
```

Comment:
this is actually needed much earlier; the only reason that the original line existed was that the slurm needs the `srun` 

so the -T opts should be built into the commands probably after parsing the options